articles/logic-apps/logic-apps-control-flow-run-steps-group-scopes.md

```diff
@@ -104,13 +104,13 @@ so save your work often.
 
       ![Set up "Bing Maps - Get route" action](./media/logic-apps-control-flow-run-steps-group-scopes/get-route.png) 
 
-      For more information about these parameters, see [Calculate a route](https://msdn.microsoft.com/library/ff701717.aspx).
+      For more information about these parameters, see [Calculate a route](/bingmaps/rest-services/routes/calculate-a-route).
```

Comment:
Thanks, please use this URL here and in the settings table: 

https://docs.microsoft.com/bingmaps/rest-services/routes/calculate-a-route